torch/lib/THPP/storages/THCStorage.hpp

```diff
@@ -21,6 +21,8 @@ struct thc_storage_traits {};
 
 template<typename real>
 struct THCStorage : public interface_traits<real>::storage_interface_type {
+  template<typename U>
```

Comment:
I believe this is the correct syntax, based on [this answer](http://stackoverflow.com/a/1810571/4256283)